dapp/src/app/(protected)/my-names/components/SubnamesPanel.tsx

```diff
@@ -44,8 +45,15 @@ export function SubnamesPanel({ selectedName, onClose }: SubnamesPanelProps) {
         }
     }, [initialNameTree]);
 
+    const { data: nameRecordData } = useNameRecord(currentNode?.name ?? '');
+    const nameRecord = nameRecordData as
+        | Extract<NameRecordData, { type: 'unavailable' }>
+        | undefined;
+
     if (!currentNode) return null;
 
+    const isExpired = nameRecord?.nameRecord ? isNameRecordExpired(nameRecord.nameRecord) : false;
```

Comment:
Shouldn't this be like: ?
```suggestion
    const isExpired = nameRecord?.nameRecord ? isGracePeriodExpired(nameRecord.nameRecord) : false;
```

Reply:
Not here because this is for disable a create subname button, which it should not be enabled when expired